docs/platforms/javascript/common/configuration/integrations/connect.mdx

```diff
@@ -22,4 +22,4 @@ The `connectIntegration` adds instrumentation for the Hapi framework to capture
 
 ## Supported Versions
 
-- Connect `^3.0.0`
+- `connect`: `^3.0.0`
```

Comment:
No init snippet here?

Reply:
yeah I need to add them for all the server frameworks, just missed this